app/models/category.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Category < ApplicationRecord
+  has_ancestry
```

Comment:
Put any `gem` stuff after `Rails` validations, associations etc.

Reply:
Got it. Corrected.